frontend/src/App.js

```diff
@@ -1,9 +1,15 @@
 import React, { Component } from 'react';
 import NavBar from './NavBar';
-import './App.css';
-
+import Home from './Home';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoggedIn: false,
+    };
+  }
+
```

Comment:
For the `console.log("Inside componentDidMount!");`, do we still need this debugging code?

Reply:
removed